src/main/kotlin/com/icat/orboarding/user/adapters/inbound/exception/ApiExceptionHandler.kt

```diff
@@ -0,0 +1,84 @@
+package com.icat.orboarding.user.adapters.inbound.exception
+
+import com.icat.orboarding.user.adapters.inbound.dtos.response.ErrorBodyResponseDTO
+import com.icat.orboarding.user.application.exceptions.CnpjAlreadyRegisteredException
+import com.icat.orboarding.user.application.exceptions.CpfAlreadyRegisteredException
+import com.icat.orboarding.user.application.exceptions.EmailAlreadyRegisteredException
+import com.icat.orboarding.user.application.exceptions.UserNotFoundException
+import org.apache.coyote.Response
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.MessageSource
+import org.springframework.context.i18n.LocaleContextHolder
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.MethodArgumentNotValidException
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import java.util.*
+
+@ControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice
```
Como estamos trabalhando com API REST, poderiamos deixar aqui com **@RestControllerAdvice**